code/src/GridAPI/Pagination.ts

```diff
@@ -60,6 +60,31 @@ namespace GridAPI.Pagination {
         );
     }
 
+    /**
+     * Gets the current page Index of the DataGrid.
+     *
+     * @export
+     * @param {string} gridID Id of the Grid from which to obtain the pagination Index
+     * @return {*}  {number} Index of the current page, 0 based. Return -1 if the Grid is not yet initialized.
+     */
+    export function GetCurrentPage(gridID: string): number {
+        PerformanceAPI.SetMark('Pagination.GetCurrentPage');
+
+        if (!OSFramework.Helper.IsGridReady(gridID)) return -1;
```

Comment:
Should we throw an error here?

Reply:
Uhm, I think that it would be too rough to throw an error...